### DISCUSSION TITLE
Fans behave differently than fan profiles.

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#404] [Q&A] [answered]
Hi, 

I am using a G15 (2022) 503RM model and I am having an issue with the fan behaving differently than the set fan profile (factory default).

According to the set fan profile (factory default), the fan should spin at 2600RPM when the CPU temperature is 60 degrees.

However, it is actually locked at 3200RPM.

screenshot -> [https://imgsh.net/a/g9BVlCR.jpg](https://imgsh.net/a/g9BVlCR.jpg)

I waited and did nothing to pinpoint the issue, but the fan speed and temperature didn't change.

But, if I set the Apply Custom Fan Curve option, it works fine. Working as 2600RPM .

This issue occur with the CPU and GPU both.

Can I select Factory Defaults and then apply the Apply Custom Fan Curve option?

Is the factory default value different from the default value entered in the bios?

Thank you.

**Comment by seerge** [comment, marked as answer]
@cheonmux Hello, 

What it shows as "factory default" - is a reading it gets either from bios or when bios doesn't return anything (for some models including g15) - it just shows a template curve (that i simply took from my own g14) :)

So threat that curve just as a template, not an actual chart, especially on your model.

**Reply by cheonmux** [reply]
Hi, @seerge 

Thank you so much for your quick response.

So that graph is not actually the factory default values for my model, just a sample.

I'll just use it without applying the Apply Custom Fan Curve.

Thanks for letting me know!

**Reply by seerge** [reply]
@cheonmux yes, there is no way to read it for your model
